app/models/label.rb

```diff
@@ -8,4 +8,8 @@ class Label < ApplicationRecord
   # scopes
   pg_search_scope :search_by_name, against: :name, using: { tsearch:
                                                             { prefix: true } }
+
+  def playlists
+    Playlist.where(id: albums.joins(:playlists).pluck('playlists.id'))
```

Comment:
I always get confused with how to do proper and efficient joins with ActiveRecord, but I do know that doing the whole `SELECT * FROM table WHERE id IN (1,2,3,4,5,6)` has its limits in mysql, and postgres I believe will build a temporary table to join with (not totally sure about that).

Especially if labels can have hundreds or thousands of albums, this might get heavy.